packages/core/src/stateUtils.ts

```diff
@@ -1606,7 +1624,22 @@ export function macrostep(
   }
 
   let nextSnapshot = snapshot;
-  const states: AnyMachineSnapshot[] = [];
+  const microstates: AnyMachineSnapshot[] = [];
+
+  function addMicrostate(
+    microstate: AnyMachineSnapshot,
+    event: AnyEventObject,
+    transitions: AnyTransitionDefinition[]
+  ) {
+    actorScope.system._sendInspectionEvent({
+      type: '@xstate.microstep',
+      actorRef: actorScope.self,
+      event,
+      snapshot: microstate,
+      transitions
```

Comment:
To err on the safe side of things I'd rename this to `_transitions`:
```suggestion
      _transitions: transitions
```

What information do you plan to use from those on the inspector's side?